### ISSUE TITLE
Too many bubble column particles

### ISSUE THREAD

**Issue by BlueHDGaming** [AViewFromTheTop/ParticleTweaks#5]
There is an issue with the bubble_column_up particles produced by magma blocks underwater, when you stand right next to the magma blocks there will be way too many particles. This is probably because they don't despawn/pop on the way down like they do in vanilla.
![2024-12-21_21 37 22](https://github.com/user-attachments/assets/bf9689c6-09e8-4db0-a42c-55100886467b)
![2024-12-21_21 44 40](https://github.com/user-attachments/assets/a5dd5c8b-d2a3-44db-a0bc-cf02e17f0b5b)

**Comment by BlueHDGaming** [comment]
This mod also is not compatible with Make Bubbles Pop mod. (bubble particles don't show at all)

**Comment by BlueHDGaming** [comment]
btw I#m in 1.21 with the newest fabric loader and api and the newest version of the mod

**Comment by Keaton42** [comment]
Can confirm. Version 1.21.1 Fabric, newest loader and api and mod, and there are so so many particles that my game gets dumped to 5x less fps and 7 tick rate

**Comment by Tschipcraft** [comment]
> This mod also is not compatible with Make Bubbles Pop mod. (bubble particles don't show at all)

Yeah, I noticed that too. Probably because I changed the way `age` works. We'll figure something out!
@AViewFromTheTop if you want, hit me up on discord (`@tschipo`). CurseForge dms are a bit hard to keep track of imo

**Comment by Yum-Muy** [comment]
Can confirm on the latest version available for fabric 1.20.1 too sadly, really love this mod and sad to have to disable it (at least for the time being, hopefully)

**Comment by Yum-Muy** [comment]
guessing no update?

**Comment by Keaton42** [comment]
Oof...

**Comment by AViewFromTheTop** [comment]
Hey, my apologies. I've had a _**ton**_ to work on. In 1.21.9 I'll probably rethink bubble particles a tad, including removing my custom behavior if Make Bubbles Pop is installed.

**Comment by AViewFromTheTop** [comment]
I personally have no interest in maintaining my "smaller" mods such as this one below the latest version, so I do apologize if this is an inconvenience for anyone.

**Comment by Keaton42** [comment]
Thank you for being transparent! I forgot I had Make Bubbles Pop installed, and I wish you well with your work on other mods! It’s frustrating having so many sub-versions in 1.21 because it’s a lot of work for modders and modpack curators!

And remember that the reason we report bugs and stay on these threads is because we love your mods! 😊
